<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>486</width>
    <height>349</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="7" column="0">
    <widget class="QRadioButton" name="radio_color_linear">
     <property name="text">
      <string>Linear</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Color:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QRadioButton" name="radio_size_linear">
     <property name="text">
      <string>Linear</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QRadioButton" name="radio_size_fixed">
     <property name="text">
      <string>Fixed</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_6">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Size</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QLineEdit" name="value_cmap_vmin"/>
   </item>
   <item row="6" column="1">
    <widget class="QColorBox" name="label_color">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="value_fixed_size"/>
   </item>
   <item row="14" column="0">
    <widget class="QLabel" name="label_4">
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Alpha:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="value_size_vmin"/>
   </item>
   <item row="6" column="0">
    <widget class="QRadioButton" name="radio_color_fixed">
     <property name="text">
      <string>Fixed</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
       <widget class="QToolButton" name="button_flip_size">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">padding: 0px</string>
        </property>
        <property name="text">
         <string>⇄</string>
        </property>
       </widget>
   </item>
   <item row="3" column="3">
    <widget class="QLineEdit" name="value_size_vmax"/>
   </item>
   <item row="2" column="1" colspan="3">
    <widget class="QComboBox" name="combo_size_attribute"/>
   </item>
   <item row="4" column="1" colspan="3">
    <widget class="QSlider" name="slider_size_scaling">
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>50</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="7" column="1" colspan="3">
    <widget class="QComboBox" name="combo_cmap_attribute"/>
   </item>
   <item row="8" column="3">
    <widget class="QLineEdit" name="value_cmap_vmax"/>
   </item>
   <item row="13" column="1" colspan="3">
    <widget class="QColormapCombo" name="combo_cmap"/>
   </item>
   <item row="14" column="1" colspan="3">
    <widget class="QSlider" name="slider_alpha">
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="8" column="2">
       <widget class="QToolButton" name="button_flip_cmap">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">padding: 0px</string>
        </property>
        <property name="text">
         <string>⇄</string>
        </property>
       </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QColorBox</class>
   <extends>QLabel</extends>
   <header>glue.utils.qt.colors</header>
  </customwidget>
  <customwidget>
   <class>QColormapCombo</class>
   <extends>QComboBox</extends>
   <header>glue.utils.qt.colors</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
